lib/reek/smell_detectors/base_detector.rb

```diff
@@ -14,7 +14,9 @@ module SmellDetectors
     # for details.
     #
     # :reek:UnusedPrivateMethod: { exclude: [ smell_warning ] }
+    # :reek:TooManyMethods: { max_methods: 18 }
```

Comment:
I hear you screaming and rightfully so. But fixing this whole mess is something I'd really like to tackle in the course of #1111